explainer.md

```diff
@@ -58,22 +58,22 @@ encrypted and congestion-controlled communication.
 
 ## Proposed solutions
 
-1. A set of generic transport mixins that can be provided by any transport,
+1. A generic transport interface that can be provided by any transport,
    but match closely with QUIC's capabilities.
 
-2. A specific transport based on QUIC that implements all of the transport mixins.
+2. The transport interface can talk [a QUIC based protocol](https://tools.ietf.org/html/draft-vvv-webtransport-quic).
 
-3. A specific transport based on HTTP/3 that allows a subset of the transport
-   mixins able to be pooled with HTTP traffic (sharing a congestion control context).
+3. The transport interface can talk [a HTTP/3 based protocol](https://tools.ietf.org/html/draft-vvv-webtransport-http3)
+   that allows web developers to reuse HTTP/3 connections (sharing a congestion control context).
 
 ## Example of sending unreliable game state to server using QUIC datagrams
 
 ```javascript
 // The app provides a way to get a serialized state to send to the server
 function getSerializedGameState() { ... }
 
-const transport = new QuicTransport('example.com', 10001);
-const datagramWriter = transport.sendDatagrams().getWriter();
+const transport = new WebTransport('quic-transport://example.com:10001/path');
+const datagramWriter = transport.writableDatagrams.getWriter();
```

Comment:
`datagramWritable` is what we've implemented in Blink.

Reply:
Done.